cove_360/lib/threesixtygiving.py

```diff
@@ -979,6 +990,180 @@ def process(self, grant, path_prefix):
         self.message = mark_safe(self.check_text['message'][self.grants_percentage])
 
 
+class ImpossibleDates(AdditionalTest):
+    """Check if dates supplied are plausible (eg no 31st Feb)"""
+
+    check_text = {
+        "heading": mark_safe("dates that don't (or won't) exist"),
```

Comment:
I'm no longer happy with this wording. 

Let's change it to "dates that didn't, or won't, exist".

Reply:
I have updated the wording.